### PR TITLE
Update typescript file. Add tslint.

### PR DESCRIPTION
**Pull request by davezabriskie**

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -1,110 +1,110 @@
...
+    return getRegexGroupFromTitle(review, /(?<=-).*/g);
 }
 
-interface Review {
-    date: string,
-    url: string,
-    artist: string,
-    album: string,
-    rating: string
+function isBlacklistedReview(review: youtube_v3.Schema$PlaylistItemSnippet) : boolean {
```
Comment (jangsi) [inline index.ts:96]: This fn name is a bit problematic

File: index.ts [hunk 2]
```diff
@@ -1,110 +1,110 @@
...
-
-    const snippets = albumReviews.map(review => ({
-        date: review.publishedAt.substring(0, 10),
-        url: `youtube.com/watch?v=${review.resourceId.videoId}`,
-        artist: getArtist(review),
-        album: getAlbum(review),
-        rating: getRating(review.description)
-    }))
+function isPresent<T>(t: T | undefined | null | void): t is T {
+    return t !== undefined && t !== null;
```
Comment (jangsi) [inline index.ts:102]: Is an empty string or zero considered "Present"?
```suggestion
    return !!t;
```

File: index.ts (hunk 1 above)
Comment (jangsi) [reply index.ts:96]: Would love to see this fn name used in a positive way too. Like if we could showcase Black artists and the associated reviews, `isBlacklistedReview` would make sense in that context.

Comment (davezabriskie) [reply index.ts]: Good call. Renamed method.

File: index.ts (hunk 2 above)
Comment (davezabriskie) [reply index.ts]: Aye, was too narrow in scope.

**Comment by davezabriskie** [comment]
Current output.

[reviews.json.txt](https://github.com/ptstory/fantano-fetcher/files/4816892/reviews.json.txt)

**Pull request merged**